test/unit/platform.spec.ts

```diff
@@ -1,44 +1,8 @@
 import assert from "assert";
-import * as os from "os";
-import * as util from "util";
-import { getSafeUserAgentSegmentInfo, getUserAgent } from "../../dist-esm/common/index";
 import { Constants } from "../../dist-esm/index";
 
 const packageJson = require("../../package.json");
 
-describe("getUserAgent", function() {
```

Comment:
probably over zealous to remove it entirely, right? Should check the method still works and returns the right versions in it, right?

Reply:
Yep. Added test for node version and SDK